pkg/controller/gittrack/gittrack_controller.go

```diff
@@ -200,7 +200,12 @@ func (r *ReconcileGitTrack) getFiles(gt *farosv1alpha1.GitTrack) (map[string]*gi
 		return nil, err
 	}
 
-	globbedSubPath := strings.TrimPrefix(gt.Spec.SubPath, "/") + "{**/*,*}.{yaml,yml,json}"
+	subPath := gt.Spec.SubPath
+	if !strings.HasSuffix(subPath, "/") {
```

Comment:
This reads a bit odd to me.
If there isn't a slash, we're adding one - only so that we can safely remove it in L208.

Wouldn't
```
if strings.HasSuffix(subPath, "/") {
  subPath = strings.TrimPrefix(subpath, "/")
}
globbedSubpath := subPath + "{**/*,*}.{yaml,yml,json}"
```
make more sense?

Reply:
Except the removal in L208 is any leading slash, wheres we're adding a trailing slash if there isn't one.